src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -50,6 +50,13 @@ public override IEnumerable<string> PreprocessorSymbolNames
                 throw new ArgumentOutOfRangeException(nameof(kind));
             }
 
+#if !SCRIPTING
+            if (kind != SourceCodeKind.Regular)
+            {
+                throw new NotSupportedException(kind.ToString());
```

Comment:
Why was this changed and is there any workaround that still allows parsing of interactive script sources?